tests/Unit/Utilities/TypeTraits/Test_GetFundamentalType.cpp

```diff
@@ -21,3 +21,17 @@ static_assert(
 static_assert(std::is_same_v<typename tt::get_fundamental_type_t<int>, int>,
               "Failed testing get_fundamental_type");
 // [get_fundamental_type]
+
+static_assert(
+    std::is_same_v<
+        typename tt::get_complex_or_fundamental_type_t<std::array<double, 2>>,
+        double>);
+static_assert(std::is_same_v<typename tt::get_complex_or_fundamental_type_t<
+                                 std::vector<std::complex<int>>>,
+                             std::complex<int>>);
```

Comment:
`std::complex` is only guaranteed to work for floating-point types.

Reply:
Changed to `double`